test/validate-token.test.js

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2010-2013 Richard Rodger */
+'use strict'
+
+var Seneca = require('seneca')
+
+var _ = require('lodash')
+
+var Lab = require('lab')
+var Code = require('code')
+var lab = exports.lab = Lab.script()
+var suite = lab.suite
+var test = lab.test
+var before = lab.before
+var expect = Code.expect
+
+var si = Seneca()
+
+if (si.version >= '2.0.0'){
+  si
+    .use(require('seneca-entity'))
+}
+si
+  .use('../user')
+
+var user1Data = {
+  nick: 'nick1',
+  email: 'nick1@example.com',
+  password: 'test1test',
+  salt: 'something',
+  active: true
+}
+
+suite('seneca-user login and token validation suite tests ', function () {
+  before({}, function (done) {
+    si.ready(function (err) {
+      if (err) return process.exit(!console.error(err))
+      done()
+    })
+  })
+
+  test('user/register test', function (done) {
```

Comment:
Is this test really needed here? I would suggest this be moved to the before hook as it's just setup, you don't need an test for it.